packages/components/src/badge/stories/index.story.tsx

```diff
@@ -13,6 +13,22 @@ const meta = {
 	title: 'Components/Containers/Badge',
 	id: 'components-badge',
 	tags: [ 'status-private' ],
+	args: {
+		children: 'Code is Poetry',
+		intent: 'default',
+		// @ts-ignore
+		'...props': null,
+	},
+	argTypes: {
+		// @ts-ignore
+		'...props': {
+			control: {
+				type: null,
+			},
```

Comment:
This is now a type error in Storybook 8. The new way is:

```suggestion
			control: false,
```